### PR TITLE
Vendor API v1.4: Add GCSE qualifications data

### PR DESCRIPTION
**Pull request by vassyz**
We need to update our Vendor API to represent the latest implementation of qualifications in our process.

Ensure we send the following fields over the API, as part of the GCSE object:

- currently_completing_qualification - stores the boolean response to the “Are you currently studying to retake your GCSE?” question.
- not_completed_explanation - stores the free text explanation if the above is false
- other_uk_qualification_type - stores the response to “Another UK qualification” - “Qualification name”

 [Vendor API v1.4: Add GCSE qualifications data](https://trello.com/c/LLVnaJb7/791-vendor-api-v14-add-gcse-qualifications-data)

Screenshot 2023-12-14 at 11 08 01

### CODE REVIEW

**Comment by vassyz** [comment]
[Vendor API v1.4: Add GCSE qualifications data](https://trello.com/c/LLVnaJb7/791-vendor-api-v14-add-gcse-qualifications-data)

**Review by inulty-dfe — APPROVED** [review]

**Review by inulty-dfe — APPROVED** [review]

File: config/vendor_api/v1.4.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  schemas:
+    Qualification:
+      type: object
+      additionalProperties: false
+      properties:
+        currently_completing_qualification:
+          type: boolean
+          description: The response to the “Are you currently studying to retake your GCSE?” question.
+          example: true
+          nullable: true
```
Comment (JR-G) [inline config/vendor_api/v1.4.yml:28]: These aren't rendering for me on the docs – unless i'm being completely blind?

Comment (vassyz) [reply config/vendor_api/v1.4.yml:28]: @JR-G should work fine now.

Comment (JR-G) [reply config/vendor_api/v1.4.yml:28]: yes – thanks!

**Review by JR-G — APPROVED** [review]

**Pull request merged**